docs/UserGuide.md

```diff
@@ -182,8 +182,8 @@ Edits the specified event from the event book.
   - Tags can be removed by typing `t/` without specifying any tags after it.
 
 **Example**:
-  - `edit 1 p/91234567 r/100` Edits the phone number and rate of the 1st person to be 91234567 and j100 respectively.
-  - `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be Betsy Crower and clears all existing tags.
+  - `edit 1 p/91234567 r/100` Edits the phone number and rate of the 1st event to be 91234567 and j100 respectively.
```

Comment:
I don't think our edit function works like this. Phone number/ Contact cannot be edited this way.

Reply:
I'll replace this line with:

`edit 1 r/100` Edits the rate of the 1st event to be 100.

Reply:
👍  Good that we catch this mistake from previous implementations through this renaming PR